dispatch/apps/api/serializers.py

```diff
@@ -38,6 +28,54 @@ class Meta:
             'image'
         )
 
+class UserSerializer(DispatchModelSerializer):
+    """
+    Serializes the User model.
+    """
+
+    email = serializers.EmailField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())]
+    )
+    person = ForeignKeyField(
+        model=Person,
+        serializer=PersonSerializer(),
+        validators=[UniqueValidator(queryset=User.objects.all())]
+    )
+    password_a = serializers.CharField(
+        required=False,
+        write_only=True,
+        validators=[PasswordValidator(confirm_field='password_b')]
+    )
+    password_b = serializers.CharField(required=False, write_only=True)
+
+    class Meta:
+        model = User
+        fields = (
+            'id',
+            'email',
+            'person',
+            'password_a',
+            'password_b',
+        )
+
+    def create(self, validated_data):
+
+        instance = User()
+        return self.update(instance, validated_data)
+
+    def update(self, instance, validated_data):
+
+        instance.email = validated_data.get('email', instance.email)
+        instance.person = validated_data.get('person', instance.person)
+
+        if validated_data.get('password_a'):
+            instance.set_password(validated_data.get('password_a'))
```

Comment:
this can likely be simplified to the faster
```py
    instance.set_password(validated_data['password_a'])
```
given that the existence of `password_a` has already been confirmed